### PR TITLE
Update .NET SDK for Mixed Reality Azure Management APIs

### PR DESCRIPTION
**Pull request by XiangyuL-Microsoft**
1. Spatial Anchors GA
2. Remote Rendering Public Preview

### CODE REVIEW

**Comment by isra-fel** [comment]
Please update the versions in csproj and AssemblyInfo.cs, as well as release notes in csproj.

**Comment by XiangyuL-Microsoft** [comment]
@erich-wang @isra-fel Resolve the comments by updating versions and release notes andadding an AssemblyInfo.cs.

File: sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Microsoft.Azure.Management.MixedReality.csproj [hunk 1]
```diff
@@ -6,18 +6,23 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.MixedReality</PackageId>
     <Description>Microsoft Azure Management Library of Mixed Reality cloud services.</Description>
-    <Version>0.9.0-preview.1</Version>
+    <Version>1.0.0.0</Version>
```
Comment (isra-fel) [inline sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Microsoft.Azure.Management.MixedReality.csproj:9]:
```suggestion
    <Version>1.0.0</Version>
```
3-digit versions are used in csproj and 4-digit is in AssemblyInfo.

Comment (XiangyuL-Microsoft) [reply sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Microsoft.Azure.Management.MixedReality.csproj]: Thank you very much for the helping, @isra-fel Yeming.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**